lightningd/opening_control.c

```diff
@@ -1227,3 +1406,13 @@ static const struct json_command fundchannel_complete_command = {
     "with {psbt}. Returns true on success, false otherwise."
 };
 AUTODATA(json_command, &fundchannel_complete_command);
+
+static const struct json_command json_commitchan_command = {
+    "emergencystubchannel",
```

Comment:
`recoverchannel`?

Reply:
Sure, I am anyways bad at naming things in general...